test/serverTest.js

```diff
@@ -404,6 +404,16 @@ describe("Running Selenium tests on Chrome Driver", function() {
       });
     });
 
+    it('Checking Jump to Speaker functionality', function(done) {
+      trackPage.jumpToSpeaker().then(function(val) {
+        assert.equal(val, true);
+        trackPage.visit('http://localhost:5000/live/preview/a@a.com/FOSSASIASummit/tracks.html');
```

Comment:
@Princu7 Are we doing tests on localhost only? I am not quite aware but still if it is, then the url that you hardcoded is fine. Otherwise you can use 'window.location.origin' + '/live/preview/.......'. Please correct me if I am wrong.